src/__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-named-as-default */
+import * as React from 'react';
+// import { IntlProvider } from 'react-intl';
```

Comment:
This is not good for the quality of your code. You should request a review when you are sure that everything is clean. Remove this if you don't need it.